tests/scaling-pipelines/scenario/signing-tr-tekton-bigbang-tr-multi-steps/setup.sh

```diff
@@ -0,0 +1,13 @@
+source scenario/common/lib.sh
+
+chains_setup_tekton_tekton_
+
+chains_stop
+
+(
+    wait_for_prs_finished "${TEST_TOTAL}"
+    chains_start
+    set_started_now
+) &
```

Comment:
FYI for bigbang scenario, you can have this code in tierdown.sh, you do not need `wait_for_prs_finished ...`